### DISCUSSION TITLE
Dependency missing , but present

### DISCUSSION THREAD

**Discussion by allenginter** [Digitalroot-Valheim/Digitalroot.Valheim.EpicLoot.Adventure.Bounties#25] [General]
The latest Bounties mod fails to load due to missing dependencies of Randy Knapp Epic Loot 10.0+ or higher
![image](https://github.com/Digitalroot-Valheim/Digitalroot.Valheim.EpicLoot.Adventure.Bounties/assets/57318835/0505d812-0c07-4d69-a3e9-c14f18baaa3e)

**Comment by Kobiashi87** [comment]
Having the same issue

**Comment by Digitalroot** [comment]
![image](https://github.com/Digitalroot-Valheim/Digitalroot.Valheim.EpicLoot.Adventure.Bounties/assets/3770289/21fb32bf-38d8-49ff-a43e-b4f4dbe5c2c3)

**Reply by Digitalroot** [reply]
![image](https://github.com/Digitalroot-Valheim/Digitalroot.Valheim.EpicLoot.Adventure.Bounties/assets/3770289/34187b88-881c-460a-8631-94847a4bcd1c)

**Comment by Digitalroot** [comment]
I'll release a version that will load with 0.9.39

**Reply by Digitalroot** [reply]
#26
